### PR TITLE
Feature/calendar setup

### PR DESCRIPTION
**Pull request by kevinhartmann23**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [x] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- adds styling to header and navbar
- corrects route paths to correctly render comp.
- Updates imports for needed imports?

## *Instructions for reviewer:*
*Where should the reviewer start?*
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
CALENDAR IS NOT FUNCTIONAL OR WORKING
need to research new package

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "CuraTour",
```
Comment (richardltyler) [inline public/manifest.json:2]: Good eye

**Review by richardltyler — APPROVED** [review]
Looks great. I appreciate all the commented out stuff for later use.

File: src/components/App.js [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
-      <NavBar/>
+      <Header/>
       <Switch>
-        <Route path="/" component={App.js}/>
+        <Route exact path="/" component={Dashboard}/>
+        <Route path="/calendar" component={Calendar} />
+        <Route path="/new-event" component={Form} />
+        {/* <Route path="/contacts" component={Contacts} /> */}
+        {/* <Route path="/event/:name" render={blah blah match what not here} /> */}
+        {/* <Route component={NotFound} /> */}
```
Comment (kristenmb) [inline src/components/App.js:20]: Thanks for adding these route notes. Definitely will make future route connections easier

**Review by kristenmb — APPROVED** [review]
Looks great so far. I think our research into a different Calendar package is the way to go. The documentation for this one was quite vague and definitely lacking in a lot of areas.

File: public/manifest.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+  "name": "CuraTour",
```
Comment (DrewBradley) [inline public/manifest.json:3]: That is our name!

File: src/components/App.css [hunk 4]
```diff
@@ -1,3 +1,10 @@
 .App {
+  height: 100vh;
   text-align: center;
+  background-color: #4D4D4D;
```
Comment (DrewBradley) [inline src/components/App.css:4]: Nice color choice! Gorgeous!

File: src/components/App.js [hunk 5]
```diff
@@ -1,20 +1,25 @@
...
 function App() {
   return (
     <div className="App">
-      <NavBar/>
+      <Header/>
       <Switch>
-        <Route path="/" component={App.js}/>
+        <Route exact path="/" component={Dashboard}/>
+        <Route path="/calendar" component={Calendar} />
+        <Route path="/new-event" component={Form} />
```
Comment (DrewBradley) [inline src/components/App.js:17]: Thanks for these reminders of what will be rendered in the remaining routes!

File: src/components/navbar/NavBar.js [hunk 6]
```diff
@@ -1,16 +1,44 @@
...
+  const [calendarButton, selectCalendar] = useState('')
+  const [addEventButton, selectAddEvent] = useState('')
+  const [contactsButton, selectContacts] = useState('')
+  
+  const toggleButtons = (active, inactive1, inactive2) => {
+    active('active')
+    inactive1('')
+    inactive2('')
+  }
+
```
Comment (DrewBradley) [inline src/components/navbar/NavBar.js:18]: Hooks looks goods!

**Review by DrewBradley — APPROVED** [review]

**Pull request merged**